### PR TITLE
[IMP] l10n: add video link for ecuador + fix other latam links

### PR DESCRIPTION
**Pull request by rps-odoo**
EDIT:
    This commit:
    - adds the link for the Ecuatorian localization app tour
    - changes the other latam app tour links as the short links embedded the
      `es_ES` language string in the URL, which should be avoided

### CODE REVIEW

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @rps-odoo! Thank you for tagging me to review this improvement! Your changes look good, and I only have two small corrections:
- need to update the base branch to `16.0`.  Be sure that when you create a PR you are doing so on the correct base branch
Screenshot 2023-09-25 at 10 41 05 AM
Screenshot 2023-09-25 at 10 41 49 AM
   
- update the commit tag to include the app/module: "[IMP] accounting/l10n_ec: adding link to app tour"

I will make these corrections now and push up my commit and approve my review.

**Review by samueljlieber — APPROVED** [review]
This improvement look good to me :)

**Review by chiaraprattico — APPROVED** [review]

File: content/applications/finance/fiscal_localizations/ecuador.rst [hunk 1]
```diff
@@ -15,7 +15,8 @@ The localization also Includes automations to easily predict the withholding tax
 each purchase invoice.
 
 .. seealso::
-   `Smart Tutorial - Localización de Ecuador <https://www.odoo.com/r/TnE>`_
+   - `App Tour - Localización de Ecuador <https://www.odoo.com/r/FWY>`_
+   - `Smart Tutorial - Localización de Ecuador <https://www.odoo.com/r/TnE>`_
```
Comment (jcs-odoo) [inline content/applications/finance/fiscal_localizations/ecuador.rst:19]:
```suggestion
   - `App Tour - Localización de Ecuador <https://www.youtube.com/watch?v=BQOXVSDeeK8>`_
   - `Smart Tutorial - Localización de Ecuador <https://www.odoo.com/slides/smart-tutorial-localizacion-de-ecuador-170>`_
```

**Review by jcs-odoo — COMMENTED** [review]
Hi @rps-odoo @samueljlieber and @chiaraprattico 

My issue here isn't that much with this PR's content, but it made me realize:
- The Smart Tutorial link isn't okay: it embeds the language string in the link `https://www.odoo.com/es_ES/slides/smart-tutorial-localizacion-de-ecuador-170`. We should avoid including the language string in links. It wasn't visible here because of the short link.
- Short links are not really necessary here since they don't take space: only the text is visible, not the URL. In this case, it is more transparent for the user to leave the full URL so they can know what they click when they hover over the link. (short links are, in my opinion, mainly useful to shorten longer links that include UTM parameters and that we have to copy-paste in full length (e.g., social media posts)).

I'll update the branch and change the other Latam localization docs as well (to remove the `es_ES` language string)

Cheers,

**Comment by jcs-odoo** [comment]
@robodoo r+

**Comment by rps-odoo** [comment]
Hi @jcs-odoo ,

I used the short link (https://www.odoo.com/r/FWY) because this is a tracked link that helps our team (Localization Experts @and-Odoo) to know how many people clicked on that link. This helps us measure the impact of our work. Can tracked links be included?

**Comment by jcs-odoo** [comment]
> Hi @jcs-odoo ,
> 
> I used the short link (https://www.odoo.com/r/FWY) because this is a tracked link that helps our team (Localization Experts @and-Odoo) to know how many people clicked on that link. This helps us measure the impact of our work. Can tracked links be included?
> 
> cc @samueljlieber
@rps-odoo 
Oh sorry!! I didn't see it! Sure!
Can you send me that link so I can fix it asap, or make a new pr?

**Comment by rps-odoo** [comment]
Hi @jcs-odoo ,

Sorry about this, I did not see your reply. The tracked link is the following: https://www.odoo.com/r/FWY. Please let me know if this can be fixed. Thank you!

cc @and-Odoo

**Pull request closed without merging**